apps/router-e2e/package.json

```
@@ -4,11 +4,14 @@
   "main": "index.js",
   "scripts": {
     "start": "expo start",
-    "start:rsc": "E2E_RSC_ENABLED=1 E2E_ROUTER_JS_ENGINE=hermes EXPO_USE_METRO_REQUIRE=1 E2E_CANARY_ENABLED=1 EXPO_USE_STATIC=single E2E_ROUTER_SRC=01-rsc EXPO_PUBLIC_USE_RSC=1 expo start",
-    "export:rsc": "E2E_RSC_ENABLED=1 E2E_ROUTER_JS_ENGINE=hermes EXPO_USE_METRO_REQUIRE=1 E2E_CANARY_ENABLED=1 EXPO_USE_STATIC=single E2E_ROUTER_SRC=01-rsc EXPO_PUBLIC_USE_RSC=1 expo export",
+    "start:01-rsc": "E2E_RSC_ENABLED=1 E2E_ROUTER_JS_ENGINE=hermes EXPO_USE_METRO_REQUIRE=1 E2E_CANARY_ENABLED=1 EXPO_USE_STATIC=single E2E_ROUTER_SRC=01-rsc EXPO_PUBLIC_USE_RSC=1 expo start",
+    "start:rsc": "E2E_RSC_ENABLED=1 E2E_ROUTER_JS_ENGINE=hermes EXPO_USE_METRO_REQUIRE=1 E2E_CANARY_ENABLED=1 EXPO_USE_STATIC=single E2E_ROUTER_SRC=rsc EXPO_PUBLIC_USE_RSC=1 expo start",
+    "export:rsc": "E2E_RSC_ENABLED=1 E2E_ROUTER_JS_ENGINE=hermes EXPO_USE_METRO_REQUIRE=1 E2E_CANARY_ENABLED=1 EXPO_USE_STATIC=single E2E_ROUTER_SRC=rsc EXPO_PUBLIC_USE_RSC=1 expo export",
+    "export:01-rsc": "E2E_RSC_ENABLED=1 E2E_ROUTER_JS_ENGINE=hermes EXPO_USE_METRO_REQUIRE=1 E2E_CANARY_ENABLED=1 EXPO_USE_STATIC=single E2E_ROUTER_SRC=01-rsc EXPO_PUBLIC_USE_RSC=1 expo export",
     "start:react-native-canary": "E2E_RSC_ENABLED=1 E2E_CANARY_ENABLED=1 E2E_ROUTER_SRC=react-native-canary expo start",
     "start:dom": "E2E_ROUTER_SRC=dom-components expo start -d",
     "dom:run:ios": "E2E_ROUTER_SRC=dom-components expo run:ios",
+    "headless:expo": "E2E_ROUTER_SRC=headless expo",
```

```suggestion
    "start:headless": "E2E_ROUTER_SRC=headless expo",
```

please match the format of all the other scripts